client/app/components/pages/GeneHome.vue

```diff
@@ -1612,18 +1612,13 @@ export default {
                 self.showLeftPanelForGenes();
               }
             } else {
-              if (self.selectedVariant == null) {
-                self.promiseSelectFirstFlaggedVariant()
-                .then(function() {
-                  self.$refs.navRef.onShowVariantsTab();
-                })
-              }
-              else if(self.launchedFromClin){
-                self.promiseSelectFirstFlaggedVariant()
-                    .then(function() {
-                      self.$refs.navRef.onShowVariantsTab();
-                    })
```

Comment:
@WilliamRichards2017  There is a check if `launchedFromClin` which is removed in this PR. Did you try this in clin to check if it did not have any side effects?